app/admin/dashboard.rb

```diff
@@ -1,32 +1,43 @@
 ActiveAdmin.register_page 'Dashboard' do
+  DISPLAY_MONTHLY_REPORT_LIMIT = 10
+
   menu priority: 1, label: proc { I18n.t('active_admin.dashboard') }
 
   content title: proc { I18n.t('active_admin.dashboard') } do
-    div class: 'blank_slate_container', id: 'dashboard_default_message' do
-      span class: 'blank_slate' do
-        span I18n.t('active_admin.dashboard_welcome.welcome')
-        small I18n.t('active_admin.dashboard_welcome.call_to_action')
+    columns do
+      Group.active.each do |group|
+        column max_width: '45%', min_width: '45%' do
+          panel "最新の月報 【#{group.name}グループ】" do
+            reports = MonthlyReport
+              .includes(:user)
+              .where(user_id: group.users)
+              .order(created_at: :desc)
+              .limit(DISPLAY_MONTHLY_REPORT_LIMIT)
```

Comment:
__[rubocop]__
* Align the operands of an expression in an assignment spanning multiple lines.

<!-- Comment by [SideCI](https://sideci.com) -->